mlos_bench/mlos_bench/tests/optimizers/toy_optimization_loop_test.py

```diff
@@ -95,7 +99,7 @@ def test_mock_optimization_loop_no_defaults(
 def test_flaml_optimization_loop(mock_env_no_noise: MockEnv, flaml_opt: MlosCoreOptimizer) -> None:
     """Toy optimization loop with mock environment and FLAML optimizer."""
     (score, tunables) = _optimize(mock_env_no_noise, flaml_opt)
-    assert score == pytest.approx(60.15, 0.01)
+    assert score == pytest.approx(60, 0.01)
```

Comment:
Why'd this change?

Reply:
I have reverted this, but there is a problem with this test case. There are some random times where it seems like the return value can change because of a seed change

Reply:
Please file a bug and submit a separate PR on that.  Let's keep unrelated changes in separate PRs.